kolibri/core/auth/management/commands/bulkimportusers.py

```diff
@@ -69,17 +70,26 @@
 INVALID_UUID = 12
 
 MESSAGES = {
-    UNEXPECTED_EXCEPTION: _("Unexpected exception [{}]: {}"),
-    TOO_LONG: _("'{}' is too long"),
-    INVALID: _("Not a valid '{}'"),
-    DUPLICATED_USERNAME: _("Duplicated Username"),
+    UNEXPECTED_EXCEPTION: pgettext_lazy(
+        "Error message that might appear when there's a programming error importing a CSV file",
+        "Unexpected error [{}]: {}",
```

Comment:
Any reason why the fuller context about possible error while importing is not part of the actual string? Nothing more frustrating and of a little usefulness for the user, than telling them there's been an _unexpected_ error. I understand that this is supposed to be a catch-all exception, but maybe something like: `Import operation was unsuccessful. Please contact the support team.` or similar?

Reply:
Yes, multiple reasons: this will appear for unknown errors. I.e. it could be anything, even a server failure, a network interruption, anything. If the reason were known we would not need to say Unexepected :)

Reply:
> multiple reasons

I totally get this, and if you say this string is not user-facing, then no need to discuss this further. :slightly_smiling_face: 

Unfortunately over the years I've seen this in user-facing messages way too many times, and I'm just venting my accumulated frustration as a user, not even as someone involved in UX copywriting.

Reply:
no, no, this one is going to be an user-facing message for sure, the problem here is that we can't know in advance the reason for the problem. When I talked about non user-facing messages I meant the context strings.

Reply:
If this **IS** an user-facing message, then we should definitely change the wording. 

_Unexpected error_ does not say anything to the user, it just contributes to their frustration. When something does not go as it is supposed to, we should provide a useful feedback and the recommended next steps to users, not present them with a vague un-actionable message.

Ideally this should collect the debug info that we already present in case of some errors in Kolibri, with the recommendations to contact support or post a message on the forums.

cc @jtamiace @khangmach

Reply:
@radinamatic it does collect the debug info and shows it exactly as you said

Reply:
Oh, so it does present the `Something went wrong` page with the copy-pastable debug info and the link to report on forums?

![error-page](https://user-images.githubusercontent.com/1457929/82047237-c021b100-96b2-11ea-8df4-7dd396b7cd79.jpg)

So where does the string _Unexpected error_ actually appear? :confused:

Reply:
It appears in the console, when executing `kolibri manage bulkimportusers....`
When using the browser you will see the screen you're attaching above.

Reply:
Ah, clarity!!! :+1: 

My apologies, I wasn't thinking of something appearing in the console as a _user-facing_ message :facepalm: 

So my next question is: are the console users who execute the command manually, and get the _Unexpected error_ for whatever reason, presented with the similar suggestion of what to do next, as are those who see the above message in the browser?

Reply:
They will see just the error traceback afterwards. Console is not too friendly , you know